packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -28,14 +30,22 @@ import {useHover, usePress} from '@react-aria/interactions';
 import {useLocale} from '@react-aria/i18n';
 import {useVisuallyHidden} from '@react-aria/visually-hidden';
 
-export function ListViewItem(props) {
+interface ListViewItemProps {
+  item: Node<any>,
+  isEmphasized: boolean,
+  dragHooks: DragHooks,
+  dropHooks: DropHooks
+}
+
+export function ListViewItem(props: ListViewItemProps) {
   let {
     item,
     isEmphasized,
-    dragHooks
+    dragHooks,
+    dropHooks
```

Comment:
Hm, I forget if there was a reason to pass the drag/drop hooks in via the props, but perhaps we can just grab it from the context instead? Worried if the ListViewItem will properly get the updated hooks due to the caching that Virtualizer does

Reply:
Definitely seems better to pass via context, good idea. I'll update other places as well